app/models/review.rb

```diff
@@ -0,0 +1,5 @@
+class Review < ActiveRecord::Base
+
+  validates_presence_of :rating
```

Comment:
I believe this is the newer Rails way:

```ruby
validates :rating, presence: true
```

Plus, you could then add on:

```ruby
validates :rating, presence: true, inclusion: (1..10).to_a
```

(Don't quote be on that inclusion syntax, but I think that's right.)